docs/docs/integrations/scim.mdx

```diff
@@ -74,13 +77,13 @@ At present, GrowthBook only supports Okta, but we are actively working on adding
 
 If a user is deprovisioned in your identity provider, they will be removed from GrowthBook. If they are re-provisioned, they will be added back to GrowthBook, and their role will reset to the organization's default role.
 
-**What if I already have users in GrowthBook?**
+**All the users from my group aren't being synced with my GrowthBook team. What's happening?**
 
-Existing users in GrowthBook will not be affected by SCIM. You can continue to manage them through the GrowthBook application as usual. If you wish to transition them to be managed by your identity provider, you can provision them through your identity provider. As long as the email matches, the existing GrowthBook user will be converted to be managed by your identity provider.
+If you notice that some users that you expect to be in your GrowthBook team from your identity provider are not being added in GrowthBook, double check that those users are assigned to the GrowthBook SCIM application in your identity provider. Only users that are both assigned and withing the group will be synced to the corresponding team.
 
-**Do you support groups?**
+**What if I already have users in GrowthBook?**
 
-Currently, we do not support groups, but we are actively working on adding group support in the future.
+Existing users in GrowthBook will not be affected by SCIM. You can continue to manage them through the GrowthBook application as usual. If you wish to transition them to be managed by your identity provider, you can provision them through your identity provider. As long as the email matches, the existing GrowthBook user will be converted to be managed by your identity provider.
 
 **Does GrowthBook follow SCIM 1.1 or 2.0 Protocol?**
 
```

Comment:
Looks good! Idk what the time-pressure is to get this PR merged, but it might be good to have a separate How-To guide on how to provision users via groups, similar to the how-to guide above?

Reply:
The guide above should cover all the setup needed to get someone to a state where they can set up push groups. Do you mean a how-to for how to set up group syncing? (Push groups can create, delete, and update groups and their memberships but it can't provision any users that aren't assigned to the app) I can link to some okta documentation if we want more details on how the push groups tab works if thats helpful

Reply:
Yea - maybe it's more Okta specific. Not needed now, but maybe we can add two additional sections to these docs for how to provision users, and how to push groups. But again, not needed now.